ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/cv-pathfinder-edge.cfg

```diff
@@ -571,6 +578,14 @@ router bgp 65000
    neighbor WAN-OVERLAY-PEERS password 7 htm4AZe9mIQOO1uiMuGgYQ==
    neighbor WAN-OVERLAY-PEERS send-community
    neighbor WAN-OVERLAY-PEERS maximum-routes 0
+   neighbor WAN-UNDERLAY-PEERS peer group
+   neighbor WAN-UNDERLAY-PEERS send-community
+   neighbor WAN-UNDERLAY-PEERS maximum-routes 12000
+   neighbor WAN-UNDERLAY-PEERS route-map RM-BGP-WAN-PEERING-IN in
+   neighbor WAN-UNDERLAY-PEERS route-map RM-BGP-WAN-PEERING-OUT out
```

Comment:
We should not use a peer-group here, since it will most likely be unique policies per neighbor.